test/dart2_suggestors/pubspec_over_react_upgrader_test.dart

```diff
@@ -93,6 +130,43 @@ main() {
         dependency: dependency,
         midVersionRange: midRangeMark,
       );
+
+      group('does not attempt to update dependency_overrides', () {
+        test('git', () {
+          defaultTestSuggestor(
+              expectedPatchCount: 0,
+              shouldDartfmtOutput: false,
+              validateContents: validatePubspecYaml,
+              input: ''
+                  'dependency_overrides:\n'
+                  '  over_react:\n'
+                  '    git:\n'
+                  '      url: git@github.com:cleandart/react-dart.git\n'
+                  '      ref: 5.0.0-wip\n',
+              expectedOutput: ''
```

Comment:
#nit You could leave the expected output off if you wanted as it defaults to expecting the input, but it's also more explicit stating the expected output, so just a thought either way 🤷‍♂ (if you wanted to remove it then naturally it could be for all four test cases)